doc/api/child_process.md

```diff
@@ -330,6 +334,21 @@ getVersion();
 function. Any input containing shell metacharacters may be used to trigger
 arbitrary command execution.**
 
+If the `AbortSignal` option is enabled, calling `.abort()` on the corresponding
```

Comment:
It's called `signal` above, right? (I guess this is what your bikeshedding note is about.)

Reply:
No I think this is just a mistake on my part unrelated to the bikeshedding?